src/pages/news.js

```diff
@@ -1,27 +1,74 @@
 import React from "react"
-import styled from "styled-components"
+import { useTranslation } from "react-i18next"
 import SEO from "@/components/templates/SEO"
 import Layout from "@components/templates/Layout"
-import { useTranslation } from "react-i18next"
 import Typography from "@material-ui/core/Typography"
-
+import { graphql } from "gatsby"
 import Box from "@material-ui/core/Box"
+import Link from "@material-ui/core/Link"
+import { BasicCard } from "@components/atoms/Card"
+import styled from "styled-components"
 
-const Container = styled(Box)`
-  margin-top: 16px;
+const NewsCard = styled(BasicCard)`
+  margin-top: 8px;
+  margin-bottom: 8px;
 `
 
-const LatestMediaNewsPage = ({ data, pageContext }) => {
+const NewsPage = props => {
+  const { data } = props
   const { t } = useTranslation()
+
+  const item = ({ node }) => {
+    return (
+      <NewsCard>
+        <Typography variant="body2" color="textPrimary">
+          {node.source} {node.date} {node.time}
+        </Typography>
+        <Typography variant="h6" color="textPrimary">
+          {node.title}
+        </Typography>
+      </NewsCard>
+    )
+  }
+
   return (
     <Layout>
-      <SEO title="LatestMediaNewsPage" />
-      <Typography variant="h4">{t("latest_media_news.title")}</Typography>
-      <Container>
-        <rssapp-list id="4rfSeuaLAKEOtTc7"></rssapp-list>
-      </Container>
+      <SEO title="NewsPage" />
+      <Typography variant="h4">{t("news.title")}</Typography>
+      <Typography variant="body2">
+        <Link
+          href="https://news.google.com/topics/CAAqJQgKIh9DQkFTRVFvSUwyMHZNR1J4T1hBU0JYcG9MVWhMS0FBUAE?hl=zh"
+          target="_blank"
+        >
+          {t("news.source_google")}
+        </Link>
+      </Typography>
+      <Typography variant="body2">
+        {t("waiting_time.last_updated")}
+        {data.allGoogleNews.edges[0].node.date} {data.allGoogleNews.edges[0].node.time}
```

Comment:
use template as well
and use lodash.get to avoid undefined

Reply:
where to put the lodash.get?